tensorflow-rl-pong/trainer/task.py

```diff
@@ -271,6 +275,10 @@ def main(args):
         '--gamma',
         type=float,
         default=0.99)
+    parser.add_argument(
+        '--beta',
```

Comment:
could you rename this argument "laziness" ?
And also see if a more self-explanatory name could be found for "gamma" ?

Reply:
gamma seems already what people use for discounted rewards.

"laziness" is probably okay, but there are other ways to specify laziness and here I just implemented it in a way that happened to work in the experiments.